platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css

```diff
@@ -98,11 +98,6 @@
 	box-shadow: var(--DOM_BUTTON_DECORATION_INTERACTION_BOX_SHADOW);
 }
 
-.EmfFormPrimaryActionSectionDiv .EmfFormSectionContentDiv:not(.invisible) .DomButton > .DomButtonIcon {
```

Comment:
This PR is completely unrelated to CSS. This change should not be here.

Reply:
I really should have seen that!
Done.